src/module/__init__.py

```
@@ -1,16 +1,16 @@
-    from .cf import register_module
-    from .color_rand import register_module
-    from .contest_manual import register_module
-    from .how_to_cook import register_module
-    from .nk import register_module
-    from .number_guess import register_module
-    from .peeper import register_module
-    from .pick_one import register_module
-    from .rand import register_module
-    from .uptime import register_module
+    from .cp.atc import register_module
+    from .cp.cf import register_module
```

Ruff (0.11.9)

Redefinition of unused `register_module` from line 3

(F811)